packages/components/addon/components/hds/form/error/index.hbs

```diff
@@ -0,0 +1,6 @@
+<div class={{this.classNames}} id={{this.id}} {{did-insert this.onInsert}} ...attributes>
+  <FlightIcon class="hds-form-error__icon" @name="alert-diamond-fill" />
```

Comment:
```suggestion
  <FlightIcon class="hds-form-error__icon" @name="alert-diamond-fill"  @title="Error: "/>
```
Suggestion: At the moment, there is not much distinction for a screen reader user between the helper text and the error message, whereas a sighted user can distinguish between the two using either color or the icon. I suggest we prefix the error message with something like 'Error: ' to help screen reader users differentiate them, but I would like to get @MelSumner's thoughts on this beforehand.

Reply:
@alex-ju all Flight icons have aria-hidden on them, so they are not presented to the user with AT at all. This was by design.

Reply:
That was supposed to be a temporary measure put in place, though. SVG implementation is simply incomplete which is why we went with the simpler route to begin with. It would be sensible to leave the decision as it stands and keep icons aria-hidden.

Reply:
@alex-ju let's pause on this for the moment, we can re-discuss this idea later when we will do a final check on the form controls before the "official" release